guides/AE2CableManagement.md

```diff
@@ -0,0 +1,71 @@
+Author: Shadoyama#4166
+
+# The Basics
+
+In this section, I'll be going over the basics of cabling and channels, just to make sure everyone understands or is on the same page as I am. If you understand the different types of cables, channels, and other basic stuff, this can be skipped.
+
+## Types of Cable
+
+This section will talk about the different types of cable and the advantages and disadvantages of using each type.
+
+Firstly, there are 4 different cables from Applied Energistics, and 2 from Ender IO. Here are the following from AE2:
```

Comment:
There are 6 types from ae2, might want to include dense covered and covered cables.

Reply:
I didn't even know that covered cables could be used, I thought they were just part of a crafting recipe. Will definitely update once I get a chance.